### PR TITLE
added translation to help navbar 

### PR DESCRIPTION
**Pull request by hiba-machfej**
1-added translation to both en and ar files
2-added it also to help navbar file
3-I also made the number as a link so on click it will direct to phone call

Screen Shot 2020-11-19 at 12 34 44 PM
Screen Shot 2020-11-19 at 12 35 28 PM

### CODE REVIEW

**Review by jennetturkkan — APPROVED** [review]

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
+  "helpNavbar": {
+    "title": "Immediate helpline:",
+    "phoneNum": "+1-340-5558-235"
```
Comment (louisrli) [inline public/locales/en/translation.json]: if this phone nums not being translated, take it out of here and move it to the code, saying "HELP_PHONE_NUMBER" as a const

notice how if i have to change the help phone number im changing it in two places.

File: src/components/HelpNavbar/index.jsx [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
-);
+const HelpNavbar = () => {
+  const { t } = useTranslation();
+  return (
+    <Navbar className="helpNavbar">
+      <Container>
+        <Col>
+          <p>{t("helpNavbar.title")}</p>
+          <img src={phoneIcon} alt="Phone icon" />
+          <a href="tel:+1-340-5558-235">{t("helpNavbar.phoneNum")}</a>
```
Comment (louisrli) [inline src/components/HelpNavbar/index.jsx]: and the phone number is copied again here, so thats 3 places you need to change if the phone number changes.

it should be one.

Comment (hiba-machfej) [reply src/components/HelpNavbar/index.jsx]: done

File: public/locales/ar/translation.json [hunk 3]
```diff
@@ -1,4 +1,7 @@
 {
+  "helpNavbar": {
+    "title": "رقم المساعدة الفورية:"
```
Comment (louisrli) [inline public/locales/ar/translation.json:3]: should this colon just be in the html?

**Review by louisrli — APPROVED** [review]

**Comment by hiba-machfej** [comment]
@Ammar-64

**Pull request merged**